scripts/consul-svc-healthy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -x
+
+
+cd /consul/scripts || exit 1
+
+LD_LIBRARY_PATH=.
+export LD_LIBRARY_PATH
+if ! [ "true" = "$(curl -s -G edgex-core-consul:8500/v1/agent/checks | /consul/scripts/jq -r '.[] | select(.ServiceName == "'"$1"'") | .Status == "passing"')" ]; then
```

Comment:
Not proper shell.  "if" runs an executable and runs the then part if the executable returns zero exit status.  Conditions like NOT (!) need to be pushed into the test / [  command.

Reply:
This is actually fine shell, it's just that ! is a command here not a condition. But if you feel strongly I can put it into the test

Reply:
This is also why we don't write things in shell :smiley:

Reply:
Hmmm.  on my system, `which [` returns `/usr/bin/[` and `which \!` returns nothing.  I can't find any documentation that it is supposed to work, but it seems to work on every version of shell I've tried.  You don't need to change it.

Reply:
I'd take a point off if you wrote that on a test, though. <grin>